selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/CheSeleniumOsioModule.java

```diff
@@ -12,16 +12,18 @@
 package org.eclipse.che.selenium.core;
 
 import com.google.inject.AbstractModule;
+import org.eclipse.che.selenium.core.client.TestAuthServiceClient;
+import org.eclipse.che.selenium.core.client.keycloak.OsioKeycloakTestAuthServiceClient;
 import org.eclipse.che.selenium.core.client.keycloak.cli.KeycloakCliCommandExecutor;
 import org.eclipse.che.selenium.core.client.keycloak.cli.OpenShiftKeycloakCliCommandExecutor;
 import org.eclipse.che.selenium.core.workspace.CheTestOpenshiftWorkspaceLogsReader;
 import org.eclipse.che.selenium.core.workspace.TestWorkspaceLogsReader;
 
-/** @author Dmytro Nochevnov */
-public class CheSeleniumOpenshiftModule extends AbstractModule {
+public class CheSeleniumOsioModule extends AbstractModule {
 
   @Override
   protected void configure() {
+    bind(TestAuthServiceClient.class).to(OsioKeycloakTestAuthServiceClient.class);
```

Comment:
It will lead to an error when we run the tests against Eclipse Che Single Use on OpenShift:
```
[ERROR] 1) A binding to org.eclipse.che.selenium.core.client.TestAuthServiceClient was already configured at org.eclipse.che.selenium.core.CheSeleniumSingleUserModule.configure(CheSeleniumSingleUserModule.java:27) (via modules: org.eclipse.che.selenium.core.CheSeleniumSuiteModule -> org.eclipse.che.selenium.core.CheSeleniumSingleUserModule).
[ERROR] at org.eclipse.che.selenium.core.CheSeleniumOpenShiftModule.configure(CheSeleniumOpenShiftModule.java:26) (via modules: org.eclipse.che.selenium.core.CheSeleniumSuiteModule -> org.eclipse.che.selenium.core.CheSeleniumOpenShiftModule)
```